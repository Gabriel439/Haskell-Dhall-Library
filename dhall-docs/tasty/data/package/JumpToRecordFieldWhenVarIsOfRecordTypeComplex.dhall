{-|
A complex example of the jump-to-definition feature on record literals.
-}
let a =
    { foo = "bar", number = +1 }

let b =
    { foo = "bar", number = +1 }

in a.foo ++ Integer/show a.number ++ b.foo ++ Integer/show b.number
