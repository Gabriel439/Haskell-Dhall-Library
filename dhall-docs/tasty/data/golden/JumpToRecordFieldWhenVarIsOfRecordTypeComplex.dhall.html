<!DOCTYPE HTML>
<html
  ><head
    ><title
    >/JumpToRecordFieldWhenVarIsOfRecordTypeComplex.dhall</title
    ><link href="index.css" type="text/css" rel="stylesheet"
    /><link href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;500;600;700&amp;family=Lato:ital,wght@0,400;0,700;1,400&amp;display=swap"
            type="text/css" rel="stylesheet"
    /><script src="index.js" type="text/javascript"
    /><meta charset="UTF-8"/></head
  ><body
    ><div class="nav-bar"
      ><img src="dhall-icon.svg" class="dhall-icon"
      /><p class="package-title"
      >test-package</p
      ><div class="nav-bar-content-divider"
      /><a id="switch-light-dark-mode" class="nav-option"
      >Switch Light/Dark Mode</a></div
    ><div class="main-container"
      ><h2 class="doc-title"
        ><span class="crumb-divider"
        >/</span
        ><a href="index.html"
        >test-package</a
        ><span class="crumb-divider"
        >/</span
        ><span href="index.html" class="title-crumb"
        >JumpToRecordFieldWhenVarIsOfRecordTypeComplex.dhall</span></h2
      ><a data-path="/JumpToRecordFieldWhenVarIsOfRecordTypeComplex.dhall"
          class="copy-to-clipboard"
        ><i
          ><small
          >Copy path to clipboard</small></i></a
      ><br
      /><div class="doc-contents"
        ><p
        >A complex example of the jump-to-definition feature on record literals.</p>
</div
      ><h3
      >Source</h3
      ><div class="source-code"
        ><pre
          >{-|<br
          />A complex example of the jump-to-definition feature on record literals.<br
          />-}<br
          />let <span id="var4-5" data-variable="var4-5"
                      class="variable-decl"
          >a</span
          > =<br
          />    { <span id="var5-7" data-variable="var5-7"
                        class="variable-decl"
          >foo</span
          > = &quot;bar&quot;, <span id="var5-20" data-variable="var5-20"
                                     class="variable-decl"
          >number</span
          > = +1 }<br
          /><br
          />let <span id="var7-5" data-variable="var7-5"
                      class="variable-decl"
          >b</span
          > =<br
          />    { <span id="var8-7" data-variable="var8-7"
                        class="variable-decl"
          >foo</span
          > = &quot;bar&quot;, <span id="var8-20" data-variable="var8-20"
                                     class="variable-decl"
          >number</span
          > = +1 }<br
          /><br
          />in <a href="#var4-5" data-variable="var4-5" class="variable-use"
          >a</a
          >.<a href="#var5-7" data-variable="var5-7" class="variable-use"
          >foo</a
          > ++ Integer/show <a href="#var4-5" data-variable="var4-5"
                               class="variable-use"
          >a</a
          >.<a href="#var5-20" data-variable="var5-20" class="variable-use"
          >number</a
          > ++ <a href="#var7-5" data-variable="var7-5" class="variable-use"
          >b</a
          >.<a href="#var8-7" data-variable="var8-7" class="variable-use"
          >foo</a
          > ++ Integer/show <a href="#var7-5" data-variable="var7-5"
                               class="variable-use"
          >b</a
          >.<a href="#var8-20" data-variable="var8-20" class="variable-use"
          >number</a
          ><br/></pre></div></div></body></html>